dart/pubspec.yaml

```diff
@@ -0,0 +1,18 @@
+name: flat_buffers
+version: 1.0.0
+description: >
+  FlatBuffers reading and writing library for Dart.  Use the flatc compiler to
+  generate Dart classes for a FlatBuffers schema, and this library to assist with
+  reading and writing the binary format.
+
+  Based on original work by Konstantin Scheglov and Paul Berry of the Dart SDK team.
+authors:
+- Dan Field <dfield@gmail.com>
+- Konstantin Scheglov
+- Paul Berry
+homepage: https://github.com/google/flatbuffers
+documentation: https://google.github.io/flatbuffers/index.html
+dependencies:
+dev_dependencies:
+  test: any
```

Comment:
It is better to specify version ranges, e.g. here
`test: ^0.12.33` (will be auto-upgraded when 0.12.34 will come out, but won't do it for 0.13.0).

`test_reflective_loader : ^0.1.4`